Pear/Cells/OnboardingTableViewCell.swift

```diff
@@ -92,22 +93,40 @@ class OnboardingTableViewCell: UITableViewCell {
         }
     }
 
-    func configure(with interest: Interest) {
+    func configure(with interest: InterestV2) {
         titleLabel.text = interest.name
-        categoriesLabel.text = interest.categories?.joined(separator: ", ") ?? nil
-        interestImageView.image = UIImage(named: interest.imageName)
+        categoriesLabel.text = interest.subtitle
+        if let interestImageUrl = URL(string: interest.imgUrl) {
+            interestImageView.kf.setImage(with: interestImageUrl)
+        }
 
         if lastShownItem != .interest {
             lastShownItem = .interest
             setupConstraints()
         }
     }
 
-    func configure(with group: Group) {
+    func configure(with group: GroupV2) {
         titleLabel.text = group.name
         categoriesLabel.text = nil
-        interestImageView.image = UIImage(named: group.imageName)
-     
+        if let groupImageUrl = URL(string: group.imgUrl) {
+            interestImageView.kf.setImage(with: groupImageUrl)
+        }
+
+        if lastShownItem != .group {
+            lastShownItem = .group
+            setupConstraints()
+        }
+    }
+
+    func configure(with talkingPoint: TalkingPointV2) {
```

Comment:
reminder we'll probably be scrapping this but we should get together and make 100% sure before committing to removing it